keyboards/argo_works/ishi/80/mk0_avr/info.json

```
@@ -0,0 +1,112 @@
+ {
+    "manufacturer": "Argo Works",
+    "keyboard_name": "Ishi 80 Mk 0",
+    "maintainer": "jasonj2232",
+    "bootloader": "caterina",
+    "diode_direction": "COL2ROW",
+    "processor": "atmega32u4",
+    "url": "https://qmk.fm/keyboards/",
+    "usb": {
+        "device_version": "0.0.1",
+        "pid": "0x4930",
+        "vid": "0x4157"
+    },
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "encoder": false
+    },
+    "matrix_pins": {
+        "cols": ["D3", "F4", "F5", "F6", "F7", "D7", "C6", "D4", "D2"],
+        "rows": ["B3", "B1", "B6", "B2", "D1", "D0", "B5", "B4", "E6"]
+    },    
+    "layouts": {
+        "LAYOUT_ishi_80": {
```

```suggestion
        "LAYOUT": {
```
(and update layout references in keymaps accordingly)